CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ Changelog for FrostFS Node
 ### Added
 ### Changed
 ### Fixed
+- Test `TestEvacuateNetwork/multiple_shards,_evacuate_many` (#2179)
```

Comment:
I don't think we should mention tests in the CHANGELOG.

Reply:
Removed.